nx.json

```diff
@@ -1,15 +1,30 @@
 {
-  "npmScope": "spartacus",
-  "tasksRunnerOptions": {
-    "default": {
-      "runner": "nx/tasks-runners/default",
-      "options": {
-        "cacheableOperations": ["build", "lint", "test", "e2e"]
-      }
-    }
-  },
   "workspaceLayout": {
     "appsDir": "projects/storefrontapp",
     "libsDir": "feature-libs"
+  },
+  "targetDefaults": {
+    "build": {
+      "cache": true
+    },
+    "lint": {
+      "cache": true
+    },
+    "test": {
+      "cache": true
+    },
+    "@nx/jest:jest": {
+      "cache": true,
+      "inputs": ["default", "^default"],
+      "options": {
+        "passWithNoTests": true
+      },
+      "configurations": {
+        "ci": {
+          "ci": true,
+          "codeCoverage": true
+        }
+      }
+    }
```

Comment:
[QUESTION]
Previous version of this file didn't contain notion of `jest`, `passWithNoTests` and configuraiton for `ci`.
Why do we need to add them now? Just curious to learn